### PR TITLE
Sass 687

### PR DESCRIPTION
**Pull request by BenCosfordHMRC**
https://jira.tools.tax.service.gov.uk/browse/SASS-687

##### Before Reviewing
- [x]  Have you pulled the branch down?
- [x]  Have you assigned yourself to the PR?
- [x]  Have you moved the task to “in review” on JIRA?
- [x]  Have you checked to ensure all dependencies are up to date?

##### Whilst Reviewing
- [x]  Have you run the tests?
- [x]  Have you run the journey tests?
- [x]  Have you looked at the JIRA story to make sure all Acceptance Criteria has been met?

##### After Reviewing
- [x]  Have you checked for merge conflicts?
- [ ]  Have you checked to make sure there are no builds in the pipeline before you merge?
- [ ]  Have you moved the task to “in pipeline” on Jira?

##### Before creating PR
- [x]  Have you run the tests?
- [x]  Have you run the journey tests?
- [x]  Have you addressed warnings where appropriate?

##### After PRs been Reviewed
- [x]  Have you checked the PR Builder passes?
- [x]  Have you checked for merge conflicts?
- [x]  Have you checked code coverage isn’t lower than previously?

### CODE REVIEW

File: project/AppDependencies.scala [hunk 1]
```diff
@@ -4,21 +4,21 @@ import sbt._
 object AppDependencies {
 
   val compile = Seq(
-    "uk.gov.hmrc"             %% "bootstrap-frontend-play-27" % "3.4.0",
-    "uk.gov.hmrc"             %% "play-frontend-hmrc"         % "0.49.0-play-27",
-    "uk.gov.hmrc"             %% "govuk-template"             % "5.65.0-play-27",
-    "uk.gov.hmrc.mongo"       %% "hmrc-mongo-play-27"         % "0.50.0"
+    "uk.gov.hmrc"             %% "bootstrap-frontend-play-28" % "5.3.0",
+    "uk.gov.hmrc"             %% "play-frontend-hmrc"         % "0.68.0-play-28",
```
Comment (lz1kwn) [inline project/AppDependencies.scala:8]: This could be 0.69.0-play-28

File: project/AppDependencies.scala [hunk 2]
```diff
@@ -4,21 +4,21 @@ import sbt._
...
+    "uk.gov.hmrc"             %% "govuk-template"             % "5.66.0-play-27",
```
Comment (lz1kwn) [inline project/AppDependencies.scala:9]: This could be 5.68.0-play-28

File: project/AppDependencies.scala (hunk 1 above)
Comment (BenCosfordHMRC) [reply project/AppDependencies.scala]: Updated cheers

File: project/AppDependencies.scala (hunk 2 above)
Comment (BenCosfordHMRC) [reply project/AppDependencies.scala]: Updated

**Pull request merged**